doc/source/whatsnew/v0.21.0.txt

```diff
@@ -566,6 +566,7 @@ Categorical
 - Bug in the categorical constructor with empty values and categories causing
   the ``.categories`` to be an empty ``Float64Index`` rather than an empty
   ``Index`` with object dtype (:issue:`17248`)
+- Bug in categorical operations `Series.cat <http://pandas.pydata.org/pandas-docs/stable/categorical.html#working-with-categories>' not preserving the original Series' name (:issue:`17509`)
```

Comment:
Sphinx has the ability to do local references (see http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations), so you can do `` :ref:`Series.cat <categorical.cat>`   `` instead of listing the actual full url. 
You only need to add a label at the specific location (put `` .. _categorical.cat: `` on the line before the title):

https://github.com/pandas-dev/pandas/blob/cbb090fac6a39d0e687a01b2ad1b7c136f5d92fc/doc/source/categorical.rst#L148-L150

Reply:
This is the expected change right?

![screenshot_2017-09-18-13-25-56-574_com android browser](https://user-images.githubusercontent.com/31629119/30533041-fbc6b006-9c74-11e7-89e8-ccb474aeb25c.png)

Reply:
yes!